src/handbook/customer/marketing/customer-stories.md

```diff
@@ -0,0 +1,118 @@
+---
+navTitle: Customer Stories
+meta:
+    title: Marketing - Customer Stories
+---
+
+# Customer Stories
+
+## Creating a story
+
+A customer story is the same as a case study, these are helpful in illustrating how Node-RED and FlowFuse can be useful for our potential customers. The process would be as follows:
+
+1. Raise an issue for the content in our private [Github repository](https://github.com/FlowFuse/customer){rel="nofollow"}.
+2. If the content is date specific create an 'all day' event on the Google Calendar [FlowFuse Publishing Schedule](https://calendar.google.com/calendar/u/0?cid=Y18yMGFjMmM5MmMwYmE0YTYwNDg4NDE1MjBmMGU2YWE0MGFhZGUxNTlkNThjZGY0ZGMwMjA0NTI4ZjFjMTcxZmQ0QGdyb3VwLmNhbGVuZGFyLmdvb2dsZS5jb20){rel="nofollow"}, include a link to the issue in the event description. If you don't have the permission to create events please ask our Google Workspace admin to give you access.
```

Comment:
I thought we stopped using that calendar a while ago in favour of https://github.com/orgs/FlowFuse/projects/18/views/1

cc @gdziuba 
```suggestion
2. If the content is date specific create an 'all day' event on the Google Calendar [FlowFuse Publishing Schedule](https://calendar.google.com/calendar/u/0?cid=Y18yMGFjMmM5MmMwYmE0YTYwNDg4NDE1MjBmMGU2YWE0MGFhZGUxNTlkNThjZGY0ZGMwMjA0NTI4ZjFjMTcxZmQ0QGdyb3VwLmNhbGVuZGFyLmdvb2dsZS5jb20){rel="nofollow"}, include a link to the issue in the event description. If you don't have the permission to create events please ask our Google Workspace admin to give you access.
```

Reply:
I'll remove it, then.

Reply:
Yes.  That is the plan that @Hasmin-AC and I came up with.
```suggestion
2. Please update and/or create an issue and schedule it within [Content Calender](https://github.com/orgs/FlowFuse/projects/21)
```

Reply:
How about this? In the previous step the issue is created in the content repository.
```suggestion
2. Add it to the [Content Calendar Project](https://github.com/orgs/FlowFuse/projects/21) and schedule it.
```